plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3979,7 +3979,8 @@ private DiskDef.DiskBus getGuestDiskModel(final String platformEmulator, boolean
             return DiskDef.DiskBus.SCSI;
         } else if (platformEmulator.contains("Ubuntu") ||
                 StringUtils.startsWithAny(platformEmulator,
-                        "Fedora", "CentOS", "Red Hat Enterprise Linux", "Debian GNU/Linux", "FreeBSD", "Oracle", "Other PV")) {
+                        "Fedora", "CentOS", "Red Hat Enterprise Linux", "Debian GNU/Linux", "FreeBSD", "Oracle",
+                        "Rocky", "AlmaLinux", "Other PV")) {
```

Comment:
should it be
```suggestion
                        "Rocky Linux", "AlmaLinux", "Other PV")) {
```
??

Reply:
Actually I was thinking about changing AlmaLinux to Alma. :-D

Since users are able to add new guest OS and mappings. Is it possible that the new guest OS does not start with "Rocky Linux" or "AlmaLinux",  for example,  "Rocky 10" or "Alma Linux 10" ?

Reply:
@DaanHoogland 
which one do you suggest ?
"Rocky Linux", "AlmaLinux"
vs
"Rocky", "Alma"
vs
"Rocky", "AlmaLinux"

Reply:
I am convinced and would not change it, though with your argument  "Rocky", "Alma" makes sense. I was asking because we recently made the version and hypervisor string improvement that led to the "Red" vs. "Red Hat Enterprise Linux" discrepency. We still use that one here. I have no strong opinion as long as it works ;)